payment/locale/fr/LC_MESSAGES/django.po

```diff
@@ -74,18 +88,31 @@ msgstr ""
 "Voici un bref rappel des conditions obligatoires de l'examen surveillé "
 "pour avoir la possibilité d'obtenir le certificat :"
 
-#: /edx/app/edxapp/fun-apps/payment/templates/payment/base.html:0
+#: /edx/app/edxapp/fun-apps/payment/templates/payment/base-abstract.html:0
 msgid "Home"
 msgstr "Accueil"
 
+#: /edx/app/edxapp/fun-apps/payment/templates/payment/list_orders.html:0
+msgid "Nothing has yet been ordered."
+msgstr "Rien n'a encore été commandé."
```

Comment:
Cette formulation est un peu bancale -- je sais qu'elle est dans la spécification, mais je pense qu'il faut malgré tout la modifier. On peut pour ça s'inspirer de ce qui est affiché dans le dashboard quand on n'est inscrit à aucun cours : "Vous n'avez encore réalisé aucune commande" et "You have not ordered anything yet".